.gitignore

```diff
@@ -67,6 +70,9 @@ target/
 # VS Code
 .vscode/
 
+# OSX
+.DS_Store
```

Comment:
We really should clean this huge gitignore file, but at the same time this makes it hard for other devs to accidentally add bad files.

@Frost what do you think?

Reply:
Alla andra projekt jag jobbat på har oftast en ganska stor gitignore incheckat - var är poängen med att städa upp den här?

Reply:
För att du ska ha en [global gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore] för dina grejer. Om jag inte har Mac, eller PyCharm, eller vad det nu mer är i denna fil, så är det här högst irrelevant. Pytest cache å andra sidan är relevant, eftersom det kommer från repot självt.

Reply:
Håller absolut inte med.

Reply:
Då får du berätta varför.